### ISSUE TITLE
Link export modal from delete-datasets modal

### ISSUE THREAD

**Issue by viddo** [CartoDB/cartodb#3847]
Once #1658 is done, the delete-dataset modal should be updated with the missing features from the original sketch file:

![delete-datasets modal](https://cloud.githubusercontent.com/assets/978461/5616429/99c0d668-9504-11e4-8707-fbb6391218de.png)
- [ ] Change cancel-button in delete-dataset modal to "export dataset" and point to this new modal
- [ ] Add original text w/ encouragement to export data before delete dataset
- [ ] If opened from delete-items dialog should return to that dialog once data is succesfully exported

**Comment by alonsogarciapablo** [comment]
There has been no activity on this issue for more several months. We are closing it. If you think this still needs to be addressed please open a new issue.
